src/pages/createOrderNewFlow.js

```diff
@@ -551,9 +551,11 @@ class CreateOrderPage extends React.Component {
       step: ['contributeAs', 'success'].includes(step) ? undefined : step,
     };
 
+    if (this.state.error) this.setState({ error: null });
```

Comment:
Maybe we should consider adding that rule to our ESLint configuration: https://eslint.org/docs/rules/curly

Reply:
I suggest using this rule as `multi-line` like StandardJS.
That lets you write small and readable single-line if's but enforces on bigger conditionals.